CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+### pg_auto_failover v1.6.3 (November 5, 2021) ###
+
+This is a bug fix release for the 1.6 series.
+
+This release also introduces a new ncurses interactive dashboard that makes
+it easier to understand the current state (and transitions) of a formation.
+The new command `pg_autoctl watch` can be used to monitor pg_auto_failover
+activity.
+
+#### Added
+* New ncurses dashboard with command pg_autoctl watch (#809)
+
+#### Changed
+* Allow setting maximum-backup-rate on create postgres step (#812)
+
+#### Fixed
+* Work around pg_replication_slot_advance xmin maintenance bug. (#815)
+* Fix "Current State" to "Reported State", and a docs cross-ref.
+* Monitor config set postgresql.pg_ctl bug fix (#818)
+* Refrain from using atexit() to clean-up our logs semaphore. (#811)
```

Comment:
Is this actually true? I believe that in the latest version of the PR we are still using `atexit`, we just use it in a different way.

Reply:
Ah yeah I could rephrase, I just copied the PR's title desc.

Reply:
Done now.